processor/routingprocessor/README.md

```diff
@@ -95,6 +95,7 @@ It is also possible to mix both the conventional routing configuration and the r
 
 - [OTTL] statements can be applied only to resource attributes.
 - Currently, it is not possible to specify the boolean statements without function invocation as the routing condition. It is required to provide the NOOP `route()` or any other supported function as part of the routing statement, see [#13545](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/13545) for more information.
+- If data is received on OTLP http server, `include_metadata` must be set to true in order to use context based routing.
```

Comment:
Do we want to mention gRPC as well?

Reply:
Hey @atoulme, it does work for grpc even without include_metadata enabled as grpc metadata is already included in the context when going through the grpc server